File: Python/find_maximum_value.py
# p67
def find_maximum_value(datas: list) -> int:
    max = datas[0]

    for data in datas:
        if max < data:
            max = data

    return max


datas = [540, -238, 190, 22, -645, 832, 777, -542, 0, 395]
result = find_maximum_value(datas)

print("最大値 = %d" % result)

# print("最大値 = %d" % max(datas))

File: TypeScript/FindMaximumValue.ts
//p67
const FindMaximumValue = (datas: Array<number>) => {
    let max = datas[0]

    for (const data of datas) {
        if (max < data) {
            max = data
        }
    }

    return max
}

const datas = [540, -238, 190, 22, -645, 832, 777, -542, 0, 395]
const result = FindMaximumValue(datas)

console.log(`最大値 = ${result}`)

// console.log(`最大値 = ${Math.max(...datas)}`)